CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the PHP cookbook.
 
+## 8.0.1 (2020-11-10)
```

Comment:
Please don't bump these in a PR, we'll do that separately.
```suggestion
## Unreleased
```

Reply:
Sorry, fixed